src/CMakeLists.txt

```diff
@@ -86,7 +86,10 @@ endif()
 target_include_directories(OpenCL-objects PRIVATE
   "${CLSPV_SOURCE_DIR}/include")
 if (CLVK_CLSPV_ONLINE_COMPILER)
-  set(OpenCL-dependencies ${OpenCL-dependencies} clspv_core)
+  set(OpenCL-dependencies ${OpenCL-dependencies} clspv_core LLVMSPIRVLib)
+  add_dependencies(OpenCL-objects intrinsics_gen)
```

Comment:
What is this for?

Reply:
I was previously getting errors similar to https://reviews.llvm.org/D64357, but it looks like after rebasing on top of https://github.com/kpet/clvk/pull/344 which updates to LLVM 15.0.0git this is no longer needed so I will remove this change.